### DISCUSSION TITLE
sideways scroll

### DISCUSSION THREAD

**Discussion by goyalyashpal** [doublecmd/doublecmd#908] [General]
how to scroll sideways in quick view panel and normal file listings (for example for long multiple columns) ?

i tried all the 8 combinations with these 3 keys: ctrl, shift, alt while scrolling... and none of these seems to work
no modifier does vertical scroll, ctrl + scroll changes font size, rest all do just vertical scroll

i searched for "↑" in [shortcuts documentation][shortcuts] for any combination with arrow keys, but couldn't find it either

[shortcuts]: https://doublecmd.github.io/doc/en/shortcuts.html#:~:text=Ctrl+-,%E2%86%91,-Open%20selected%20directory

**Comment by alexx2000** [comment]
Do you mean horizontal scroll?

**Reply by goyalyashpal** [reply]
> horizonatl scroll

yes

**Comment by ziki77** [comment]
I'm quite sure that the horizontal scrolling worked (with some bug) in the past with right and left keys...but now it seems not

**Comment by ziki77** [comment]
Ah, OK...one of the bug it that it works only if the window is not maximized...try right and left arrow reducing the window size. Other bug is that you can scroll left only when you reach the right scroll limit.
